utils/config-utils.js

```diff
@@ -9,7 +9,8 @@ export function initCuttleGlobals(app) {
   const cuttle = {
     version,
     // Expose app for debugging/testing in lower envs
-    app: test ? app : null,
+    // app: test ? app : null,
+    app,
```

Comment:
Why this change? Was this just for local development? I don't think we want to add `app` to the window in prod

Reply:
Oops, sorry! Forgot I was testing this locally to make sure i18n was in the global scope.